src/components/Drawer/AddReportDrawer/index.tsx

```diff
@@ -72,11 +72,16 @@ const AddReportDrawer: React.FC = () => {
 
   const handleSetLatLng = () => {
     const center = maps.getCenter();
-    if (!center) {
+    const north = maps.getBounds()?.getNorthEast().lat();
+    const south = maps.getBounds()?.getSouthWest().lat();
+    if (!center || !north || !south) {
       throw new Error("LatLng not found");
     }
 
-    const latlng = { latitude: center.lat(), longitude: center.lng() };
+    const latlng = {
+      latitude: north - (north - south) / 4,
```

Comment:
```suggestion
      latitude: south + (north - south) * 0.75,
```
Will this be better to understand? Is no just ignore it !

Reply:
我覺得...都行？
不然就改成你那樣好了哈哈哈